.gitlab-ci.yml

```diff
@@ -1,36 +1,36 @@
 stages:
+  - check
   - test
   - build
 
-clippy:
+check:
   image: paritytech/ci-linux:9575dfcd-20210729
-  stage: test
+  stage: check
   timeout: 1 hours
   script:
     - rustup component add clippy --toolchain nightly
-    - cargo +nightly clippy --all-features --all-targets -- -D warnings
+    - cargo +nightly clippy --workspace --all-targets --no-default-features -- -D warnings
+    - cargo +nightly clippy --workspace --all-targets --features try-runtime -- -D warnings
+    - cargo +nightly clippy --workspace --all-targets --features runtime-benchmarks -- -D warnings
+    - cargo +nightly clippy --workspace --all-targets --features mock -- -D warnings
+    - cargo +nightly clippy --workspace --all-targets --features std -- -D warnings
+    - cargo +nightly clippy --workspace --all-targets --all-features -- -D warnings
```

Comment:
So we mostly miss unused import warnings. And the imports are only unused in very specific feature combinations? The disadvantages of a slow CI clearly outweigh the gains from me here. 😕